bash_completion.sh

```diff
@@ -221,7 +221,7 @@ _buildtest ()
       ;;
 
     buildspec|bc)
-      local cmds="-h --help edit-test edit-file find maintainers show summary validate"
+      local cmds="-h --help edit-test edit-file find maintainers show show-fail summary validate"
```

Comment:
Please add another bash function that returns a list of failed tests

Shown below is an example 
```
(buildtest)  ~/Documents/github/buildtest/ [devel*] buildtest rt --failure --format name --terse --no-header
returncode_list_mismatch
exit1_fail
```

You will want to call this as 

```
      show-fail)
        COMPREPLY=( $( compgen -W "$(_failed_tests)" -- $cur ) );;
```